microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -0,0 +1,606 @@
+---
+title: Working with local content in WebView2 apps
+description: Ways to work with local content in WebView2 apps, including navigating to a file URL, navigating to a string, virtual host name mapping, or the WebResourceRequested event.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 01/10/2023
+---
+# Working with local content in WebView2 apps
+
+In addition to loading remote content, content can also be loaded locally into WebView2.  There are several different approaches that can be used to load local content into a WebView2 control, including: 
+* Navigating to a file URL.
+* Navigating to an HTML string.
+* Virtual host name mapping.
+* Handling the `WebResourceRequested` event.
+
+These approaches are described below.
+
+
+<!-- ====================================================================== -->
+## Navigate to a file URL
+
+WebView2 allows navigations to file URLs, to load basic HTML or a PDF.  This is the most efficient and simples approach that satisfies the basic scenario.  However, it is less flexible than the other approaches.  This is the simplest approach to load local content.  However, like the browser, file URLs are limited in some capabilities. The document origin will be `null` for a file URL.  For each resource, the full path is needed to be specified.
+
+File URLs behave like they do in the browser.  For example, you can't make an `XMLHttpRequest` (XHR) in a file URL, because you're not working in the context of a webpage.  If the WebView2 control is loading local content, still came from a remote source (such as Bing.com), versus a file that came from local.
```

Comment:
```suggestion
File URLs behave like they do in the browser.  For example, you can't make an `XMLHttpRequest` (XHR) in a file URL, because you're not working in the context of a webpage.  If the WebView2 control is loading local content, still came from a remote source (such as Bing.com), versus a file that came from the local drive.
```
@yildirimcagri - is "the local drive" ok/correct?

Reply:
I changed it to "the local drive".